lib/tasks/copy_database_to_s3.rake

```diff
@@ -38,7 +38,7 @@ namespace :scihist do
         storage_class: "STANDARD_IA",
         metadata: { "backup_time" => Time.now.utc.to_s}
       ) do |write_stream|
-      cmd.run('pg_dump', '-w', '--clean', ENV['DATABASE_URL']) do |out, err|
+      cmd.run('pg_dump', '--no-owner', '--no-acl', '--clean', ENV['DATABASE_URL']) do |out, err|
```

Comment:
What does the `-w` that we're dropping do? Looking it up myself, it seems to be a synonym for `--no-password, Never issue a password prompt. `? Do we still want the `-w` there then, it doens't seem like it's related to adding --no-owner --no-acl? https://www.postgresql.org/docs/12/app-pgdump.html

Reply:
Oh good. Yeah, I'll add it back in and (this time) document it. I looked for that -w and could not find it anywhere in the documentation, so I decided I'd rather remove it now while we're actively testing backups. As long as we know exactly what it does, we can keep it.